CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 - Fix path to Sinon in browser environments
 
+# [5.0.0](https://github.com/goodeggs/goodeggs-test-helpers/compare/v4.0.0...v5.0.0)
+
+### BREAKING CHANGES
+- Sinon no longer included. `yarn add sinon@latest` and change `import {sinon} from 'goodeggs-test-helpers'` to `import sinon from 'sinon'`
+- `useSinonSandbox()` removed. Use sinon directly:
+For stubbing, spying, and stubbing logger:
+```
+import sinon from 'sinon';
+import {stubLogger} from 'goodeggs-test-helpers';
+
+describe('sandbox test', function () {
+  const sandbox = sinon.createSandbox();
+  afterEach('restore sandbox', () => sandbox.restore());
+
+  it('does a thing', function () {
+    sandbox.stub(foo, 'bar');
+    sandbox.spy(foo, 'baz');
+    stubLogger(logger, sandbox);
+  });
+});
+```
+
+And for mocking time:
+```
+import sinon from 'sinon';
+
+describe('clock test', function () {
+  const clock = sinon.useFakeTimers({now: 0, toFake: ['Date', 'setInterval']});
```

Comment:
How do you feel about exposing this as a helper?

```js
import {setupFakeTimers} from 'goodeggs-test-helpers';

// ...

const clock = setupFakeTimers(sinon);
```

I'm not sure it gives us that much, but if we really only ever mock Date and setInterval out then maybe it's worth it? /shrug

Reply:
If we like the behavior, I could also see this helper (or a derivative helper) setting up `beforeEach/afterEach` hooks, but I feel lukewarm about it.

Personally, I feel like setting it up myself gives me less mysterious behavior in my tests and I generally feel like testing behavior should be as straightforward as possible, to the point where I'm willing to give up some DRY for it.

Reply:
Relatedly: If we ever decide to try out Ava or some other test runner, the more we couple ourselves to the mocha interface the less useful these utilities will be. So I definitely fall on the side of composing more specific helpers out of many small, less-opinionated helpers so that when the opinionated helper doesn't fit your use case, you can still make use of its composite parts.

Reply:
See new interface. This is much cleaner now!